floss/main.py

```diff
@@ -148,79 +180,95 @@ def make_parser(argv):
         "-q", "--quiet", action="store_true", help="disable all status output except fatal errors"
     )
 
-    if "-x" in argv:
-        analysis_group = parser.add_argument_group("analysis arguments")
-        analysis_group.add_argument(
-            "--minimum-length",
-            dest="min_length",
-            default=DEFAULT_MIN_LENGTH,
-            help="minimum string length",
-        )
+    analysis_group = parser.add_argument_group("analysis arguments")
+    analysis_group.add_argument(
+        "-n",
+        "--minimum-length",
+        dest="min_length",
+        default=DEFAULT_MIN_LENGTH,
+        help="minimum string length" if expert else argparse.SUPPRESS,
```

Comment:
`argparse.SUPPRESS` hides the help message in the "non-expert" mode, but the option can still be used